stdlib/cmake/modules/AddSwiftStdlib.cmake

```
@@ -895,11 +895,48 @@ endfunction()
 
 # Add a new Swift target library.
 #
+# NOTE: This function inherits all options from add_swift_target_library_single_sdk
+# except for SDK.
+# Usage:
+#   add_swift_target_library(name
+#     [TARGET_SDKS sdk1...]
+#     [add_swift_target_library_single_sdk's options ...])
+#
+# TARGET_SDKS
+#   The set of SDKs in which this library is included. If empty, the library
+#   is included in all SDKs.
+function(add_swift_target_library name)
+  set(SWIFTLIB_multiple_parameter_options TARGET_SDKS)
+  cmake_parse_arguments(SWIFTLIB
+                        ""
+                        ""
+                        "${SWIFTLIB_multiple_parameter_options}"
+                        ${ARGN})
```

Please use variables to make this easier to understand.  Please use an initialism to make the options standout.
```suggestion
  set(options)
  set(single_parameter_options)
  set(multiple_parameter_options TARGET_SDKS)
  cmake_parse_arguments(ASTL
    "${options}"
    "${single_parameter_options}"
    "${multiple_parameter_options}"
    ${ARGN})
```